### PR TITLE
Add dvorak grid option

### PR DESCRIPTION
**Pull request by floomby**
Added the option for dvorak in the grid selection to remove the need for setting a custom grid. Additionally to facilitate the change special character support was added for the predefined grids.

### CODE REVIEW

File: LuaUI/Widgets/gui_chili_integral_menu.lua [hunk 1]
```diff
@@ -192,6 +193,7 @@ options = {
 			{name = 'QWERTY (standard)',key = 'qwerty', hotkey = nil},
 			{name = 'QWERTZ (central Europe)', key = 'qwertz', hotkey = nil},
 			{name = 'AZERTY (France)', key = 'azerty', hotkey = nil},
+			{name = 'DVORAK (standard)', key = 'dvorak', hotkey = nil},
```
Comment (sprunk) [inline LuaUI/Widgets/gui_chili_integral_menu.lua:196]:
```suggestion
			{name = 'Dvorak (standard)', key = 'dvorak', hotkey = nil},
```

File: LuaUI/Widgets/gui_chili_keyboardmenu.lua [hunk 2]
```diff
@@ -154,6 +154,7 @@ options = {
 			{key='qwerty', name='QWERTY', },
 			{key='qwertz', name='QWERTZ', },
 			{key='azerty', name='AZERTY', },
+			{key='dvorak', name='DVORAK', },
```
Comment (sprunk) [inline LuaUI/Widgets/gui_chili_keyboardmenu.lua:157]:
```suggestion
			{key='dvorak', name='Dvorak', },
```

**Review by sprunk — COMMENTED** [review]
Looks fine

File: LuaUI/Widgets/gui_chili_keyboardmenu.lua (hunk 2 above)
Comment (floomby) [reply LuaUI/Widgets/gui_chili_keyboardmenu.lua:157]: Your right that all caps is probably not what is wanted. Do you prefer that I amend the original commit, or just commit the suggestions from the github ui and let you squash when you merge?

Comment (sprunk) [reply LuaUI/Widgets/gui_chili_keyboardmenu.lua:157]: I don't mind either way.

**Comment by sprunk** [comment]
Thanks!

**Pull request merged**